inference-engine/thirdparty/clDNN/runtime/CMakeLists.txt

```diff
@@ -61,6 +61,13 @@ target_link_libraries("${CLDNN_BUILD__PROJ}" PRIVATE
     openvino::itt
   )
 
+if(ENABLE_ONEDNN_FOR_GPU)
+  add_dependencies("${CLDNN_BUILD__PROJ}" onednn_gpu_build)
+  target_link_libraries("${CLDNN_BUILD__PROJ}" PUBLIC ${ONEDNN_LIBRARY})
```

Comment:
The variable defined in `thirdparty/CMakeLists.txt` is not visible here:
- `ONEDNN_INCLUDE_DIRS `
- `ONEDNN_LIBRARY`

Reply:
Why? These variables are set in parent scope (thirdparty/CMakeLists.txt), and the child scope (clDNN/runtime/CMakeLists.txt) is supposed to inherit all parent variables. Also, I tried to print the values of `ONEDNN_INCLUDE_DIRS` and `ONEDNN_LIBRARY` from clDNN/runtime/CMakeLists.txt and they looked good.

Reply:
oh, I see, thanks!